### PR TITLE
add base styles

### PR DESCRIPTION
**Pull request by nils-keller-dev**

### CODE REVIEW

**Comment by nils-keller-dev** [comment]
[setup basic html](https://trello.com/c/wk1z8n3N/4-setup-basic-html)

File: src/css/styles.css
Comment (sebakocz) [inline src/css/styles.css:1]: dark mode looks wacky imo, but won't stop me from merging

image

Comment (nils-keller-dev) [reply src/css/styles.css:1]: works on my machine
image

Comment (nils-keller-dev) [reply src/css/styles.css:1]: how did you activate the dark theme?

Comment (sebakocz) [reply src/css/styles.css:1]: I used funny plugin

https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh?hl=en

Comment (nils-keller-dev) [reply src/css/styles.css:1]: won't fix

**Pull request merged**